VimeoNetworking/VimeoNetworking/NSError+Extensions.swift

```diff
@@ -213,6 +213,25 @@ public extension NSError
         return nil
     }
     
+        /// Returns the invalid parameters of the failing response, if available
+    public var vimeoInvalidParametersErrorCodes: [Int]
```

Comment:
I think this code also exists in VimeoUpload. I suspect there's a bunch of redundancy between the two libs. Future work.